client/my-sites/customer-home/cards/features/site-preview/style.scss

```diff
@@ -32,22 +32,6 @@
 			height: 235px;
 			pointer-events: none;
 			width: 100%;
-			iframe {
```

Comment:
I think removing these styles is causing the iframe to look like this now: 

<img width="363" alt="Screenshot 2024-02-20 at 11 58 34 AM" src="https://github.com/Automattic/wp-calypso/assets/2124984/40fa1398-13d2-4a8d-903b-eb72e5270b08">

Reply:
Sorry - forgot to re-add those styles when I removed the thumbnail! Should be all good now.